<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAgingTest</class>
 <widget class="QDialog" name="DialogAgingTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>老化测试</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1131</width>
     <height>711</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>730</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewReg">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>201</width>
     <height>711</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>730</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>730</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0000-00-00 00:00:00.000</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSave">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>730</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>保存到文件</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGoto">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>730</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件 ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFile">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>730</y>
     <width>461</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>730</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>文件:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
